src/types/phone.js

```diff
@@ -0,0 +1,22 @@
+const phone = {
+  default: context => {
+    const defaultPhoneRegEx = /^\(?([0-9]{3})\)?[-. ]?([0-9]{3})[-. ]?([0-9]{4})$/
+    if (!defaultPhoneRegEx.test(context.value)) {
+      context.fail('Value must be a valid phone number')
+    }
+  },
+  international: context => {
+    const intlPhoneRegEx = /^\+?([0-9]{2})\)?[-. ]?([0-9]{4})[-. ]?([0-9]{4})$/
+    if (!intlPhoneRegEx.test(context.value)) {
+      context.fail('Value must be a valid phone number')
+    }
+  },
+  numeric: context => {
+    const numericPhoneRegEx = /\d{7,10}/
```

Comment:
This allows 7 _through_ 10 digits, not 7 _or_ 10 digits (also, the readme says 8-10 for this type)